### PR TITLE
upgrade Zapier & Mocha

### PR DESCRIPTION
**Pull request by fharper**

### CODE REVIEW

File: test/mindee.js [hunk 1]
```diff
@@ -27,10 +27,9 @@ describe('Create - Mindee_invoice', () => {
       inputData: {},
     };
 
-    const result = await appTester(
```
Comment (AbdelfattahSekak) [inline test/mindee.js:30]: I don't know why you had to change the promise from the new syntax (async await) to the old syntax (then catch). Is it necessary ?

**Review by AbdelfattahSekak — COMMENTED** [review]
I don't know why you had to change the promise from the new

File: test/mindee.js (hunk 1 above)
Comment (fharper) [reply test/mindee.js:30]: There is something with Moca & Zapier Testing Class, it was giving me a deprecated error (not a warning)

`[DEP0097] DeprecationWarning: Using a domain property in MakeCallback is deprecated. Use the async_context variant of MakeCallback or the AsyncResource class instead. (Triggered by calling processImmediate on process.)`

Comment (fharper) [reply test/mindee.js:30]: A change since Node.js 14.something

Comment (AbdelfattahSekak) [reply test/mindee.js:30]: Okey, I m not sure why but you can keep the old syntax if it solve the deprecation issue

Comment (fharper) [reply test/mindee.js:30]: Me neither!

**Pull request merged**